README.md

```diff
@@ -2,4 +2,5 @@
 Add your ideas below!
 
 * Wobbly physics
-* Make a page that includes a timeline of how far the website has gone
\ No newline at end of file
+* Make a page that includes a timeline of how far the website has gone
+* A recap of our meetings / VoD / Transcript for future group members to see
```

Comment:
Jonathan please add periods to all of the sentences

Reply:
working on it